Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Multistage docker build, requires docker 17.05
+
+# builder stage
+FROM rust:1.26 as builder
+
+RUN set -ex && \
+    apt-get update && \
+    apt-get --no-install-recommends --yes install \
+        clang \
+        libclang-dev \
+        llvm-dev \
+        libncurses5 \
+        libncursesw5 \
+        cmake \
+        git
+
+WORKDIR /usr/src/grin
+
+# Copying Grin
+COPY . .
+
+# Building Grin
+RUN cargo build --release
+
+# runtime stage
+FROM debian:9.4
+
+RUN set -ex && \
+    apt-get update && \
+    apt-get --no-install-recommends --yes install locales && \
+    apt-get clean && \
+    rm -rf /var/lib/apt && \
```

Comment:
Just curious, why remove all apt data here? Smaller image file?

Reply:
Yes, however I'm updating this part and we will not need to remove the apt files.